edalize/yosys.py

```diff
@@ -32,6 +32,9 @@ def get_doc(cls, api_ver):
                         {'name' : 'yosys_synth_options',
                          'type' : 'String',
                          'desc' : 'Additional options for the synth command'},
+                        {'name' : 'yosys_additional_commands',
```

Comment:
I prefer additional commands to come in as tcl files instead as is done in the other tcl-supporting backends

Reply:
Done.